hadoop/src/main/java/org/apache/carbondata/hadoop/api/CarbonInputFormat.java

```diff
@@ -472,9 +471,6 @@ public IndexFilter getFilterPredicates(Configuration configuration) {
     QueryStatisticsRecorder recorder = CarbonTimeStatisticsFactory.createDriverRecorder();
     QueryStatistic statistic = new QueryStatistic();
 
-    // get tokens for all the required FileSystem for table path
```

Comment:
@ Reviewers: let me know if any impact by remove this call. Removed and tested in closer, didn't find any problem. 
so, is it required ? is this API call is responsible for renewing tokens ?

Reply:
Also removed and tested in user environment, no issues observed